.github/workflows/format.yml

```
@@ -14,14 +14,14 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
-    - uses: actions/setup-python@v5
+    - name: Set up the latest version of uv
+      uses: astral-sh/setup-uv@v5
       with:
-        python-version: 3.12
+        enable-cache: true
+        cache-dependency-glob: "pyproject.toml"
```

[must?] This should be 

```suggestion
        cache-dependency-glob: "uv.lock"
```

?

now it seems to use old cache even if we've re-lock.

----

in `.github/workflows/publish.yml`, `.github/workflows/test.yml` as well.

----

[astral-sh/setup-uv@v5
](https://github.com/astral-sh/setup-uv?tab=readme-ov-file#cache-dependency-glob)

> The default is

```
cache-dependency-glob: |
  **/requirements*.txt
  **/uv.lock
```

it seems fine even if we don't set `cache-dependency-glob:` explicitly

---

> it seems fine even if we don't set cache-dependency-glob: explicitly
Oh, I didn't know that. I agree with this idea 👍

---

fixed in [349cded](https://github.com/m3dev/gokart/pull/429/commits/349cdedd9e0bed4db1c0276a97526bfd1c2e99cd)